pages/index.tsx

```diff
@@ -1,9 +1,21 @@
-import type { NextPage } from "next";
+import type { NextPage, NextPageContext } from "next";
 import Head from "next/head";
-import { useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import { useEffect, useLayoutEffect, useState } from "react";
+import useApp from "../hooks/useApp";
+
+
 
 const Index: NextPage = () => {
   const [isBrowser, setIsBrowser] = useState(false);
+  const router = useRouter()
+  const appState = useApp()?.getState();
+
+  useEffect(() => {
+    if (appState?.domain && isBrowser) {
+      router.push("/configuration", { query: location.search })
```

Comment:
do we want it to be redirected here, or just moved under `/`?

Reply:
We want to be redirected to `/configuration`. `/` is just the Index page (information for developers), configuration is the page that should be displayed in Saleor Dashboard.
@magul did you meant to use `router.replace` instead of `router.push`?

Reply:
@witoszekdev I think that's a good approach, `/start-up` would be the "you did it!" view. @magul @zaiste what do you think?

Reply:
Changed it to `router.replace` - if you clicked back button in dashboard, you'd have a navigation in iframe instead in dashboard

Reply:
I was thinking about not using `router` at all here, but I don't have super strong opinion about that.